.github/workflows/release.yml

```diff
@@ -0,0 +1,28 @@
+name: "Release"
+
+on:
+  push:
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+" # Push events to matching v*, i.e. v1.0, v20.15.10
+
+jobs:
+  goreleaser:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.15
+
+      - name: Run GoReleaser
+        uses: goreleaser/goreleaser-action@v2
+        with:
+          version: latest
+          args: release --rm-dist --release-notes=release_notes.md
```

Comment:
Here I am adding a markdown file that can contain any info we would like to relay about the release. Any data placed in here will be put into the body of the release. If we would like to have the same message each time, like we do now, this will not be required.

Reply:
Can we continue to link to the changelog? (Also, I think the message is maybe not exactly the same every time - technically it links to the specific link in the changelog?)

Reply:
yes, just need to work out how to create the needed link. getting odd dirty git errors with the approaches I have tried.

Reply:
I have it appearing like so: https://github.com/marbar3778/tendermint/releases/tag/v1.4.7. Our naming template will be `1.4.7 (WARNING: BETA SOFTWARE)`